Topic4/src/interfaces/Item.java

```diff
@@ -0,0 +1,18 @@
+package interfaces;

```

Comment:
The package should have a more descriptive name not only interfaces, something related with the service that you are creating/modelling.
In fact, in this case, they should be located in the same package that the classes implementing them